docs/how-to/make-a-standard-detector.rst

```diff
@@ -0,0 +1,44 @@
+.. note::
+
+   Ophyd async is included on a provisional basis until the v1.0 release and 
+   may change API on minor release numbers before then
+
+Make a StandardDetector
+=======================
+
+`StandardDetector` is an abstract class to assist in creating devices to control EPICS AreaDetector implementations.
+The `StandardDetector` is a simple compound device, with 2 standard components: 
+
+- `DetectorWriter` to handle data persistence, i/o and pass information about data to the RunEngine (usually an instance of :py:class:`HDFWriter`)
+- `DetectorControl` with logic for arming and disarming the detector. This will be unique to the StandardDetector implementation.
+
+These standard components are not devices, and therefore not subdevices of the `StandardDetector`, typically they are enabled by the use of two other components which are:
+
+- An implementation of :py:class:`NDPluginBase`, an entity object mapping to an AreaDetector NDPluginFile instance (for :py:class:`HDFWriter` an instance of :py:class:`NDFileHDF`)
+- :py:class:`ADBase`, or an class which extends it, an entity object mapping to an AreaDetector "NDArray" for the "driver" of the detector implementation
+
+Writing a StandardDetector implementation
+-----------------------------------------
+
+Define a :py:class:`FooDriver` if the NDArray requires fields in addition to those on :py:class:`ADBase` to be exposed. It should extend :py:class:`ADBase`.
```

Comment:
Based on the above, would it be worth splitting this into two sections, `areaDetector` Driver/controller classes, and non-`areaDetector` ones? I imagine integration of AD detectors will be more common, but may be worth highlighting a more general approach as well?

Reply:
Yes, as we're going to have Odin and PandA as well as areaDetector...